cfgov/login/migrations/0005_add_default_user_group.py

```diff
@@ -0,0 +1,36 @@
+# Generated by Django 4.2.11 on 2024-05-20 18:18
+
+from django.db import migrations
+from django.conf import settings
+
+
+def forwards(apps, schema_editor):
```

Comment:
Even though I think I suggested moving this to a data migration, I'm having second thoughts. Doing it this way means we won't be able to squash down these migrations to get rid of this. Should this instead live in the initial_data script? This gets run on every deploy.